src/libs/Clipboard/index.js

```diff
@@ -1,9 +1,48 @@
 // on Web/desktop this import will be replaced with `react-native-web`
 import {Clipboard} from 'react-native-web';
 import lodashGet from 'lodash/get';
+import CONST from '../../CONST';
+import * as Browser from '../Browser';
 
 const canSetHtml = () => lodashGet(navigator, 'clipboard.write');
 
+/**
+ * Deprecated method to write the content as HTML to clipboard.
+ * @param {String} html HTML representation
+ * @param {String} text Plain text representation
+ */
+function setHTMLSync(html, text) {
+    const node = document.createElement('span');
+    node.textContent = html;
+    node.style.all = 'unset';
+    node.style.opacity = '0';
+    node.style.position = 'absolute';
+    node.style.whiteSpace = 'pre-wrap';
+    node.style.userSelect = 'text';
+    node.addEventListener('copy', (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+        e.clipboardData.clearData();
+        e.clipboardData.setData('text/html', html);
+        e.clipboardData.setData('text/plain', text);
+    });
+    document.body.appendChild(node);
+
+    const selection = window.getSelection();
+    selection.removeAllRanges();
+    const range = document.createRange();
+    range.selectNodeContents(node);
+    selection.addRange(range);
+
+    try {
+        document.execCommand('copy');
+        // eslint-disable-next-line no-empty
+    } catch (e) {}
```

Comment:
Any specific reason why we need an empty catch here?

Reply:
Got it! Can you add a comment explaining why then?

Reply:
Can you move the comment inside the catch as discussed here? https://expensify.slack.com/archives/C01GTK53T8Q/p1670607994265879